src/include/wiredtiger.in

```diff
@@ -921,6 +921,12 @@ struct __wt_session {
 	 * cache.  Not compatible with LSM tables; see @ref
 	 * tuning_cache_resident for more information., a boolean flag; default
 	 * \c false.}
+	 * @config{log = (, the transaction log configuration for this object.
```

Comment:
There is no change to any current behavior or default setting.  We don't document most API changes in there.  So my inclination is that it does not.  Let me know if you disagree.